app/controllers/listings_controller.rb

```diff
@@ -107,4 +107,9 @@ def check_conflicts(check_in_date, check_out_date, listing)
     check = @listing.bookings.where('? < check_in_date AND check_out_date < ?', check_in_date, check_out_date)
     check.size > 0 ? true : false
   end
+
+  def address_changed?
+    @listing = Listing.find(params[:id])
+    @listing.address != params[:address]? true : false 
+  end
```

Comment:
I don't see this method being used anywhere in your app.
This is not at all connected to the google maps method `:address_changed?` in the `listing.rb`